tests/study/storage/variantstudy/test_variant_study_service.py

```diff
@@ -212,12 +212,13 @@ def test_generate_task(
             study_version=study_version,
         )
 
-        execute_or_add_commands(
-            variant_study,
-            file_study,
-            commands=[create_area_fr, create_st_storage],
-            storage_service=study_storage_service,
-        )
+        with patch("antarest.study.business.utils.get_current_user", return_value=DEFAULT_ADMIN_USER):
```

Comment:
I think we can replace it with the new `current_user_context` method, it will be more reliable.